src/dashboard/src/main/management/commands/rebuild_transfer_backlog.py

```diff
@@ -70,11 +77,17 @@ def add_arguments(self, parser):
             "--transfer-backlog-dir", default=self.DEFAULT_TRANSFER_BACKLOG_DIR
         )
         parser.add_argument("--no-prompt", action="store_true")
+        parser.add_argument("--reindex-from-storage-service", action="store_true")
```

Comment:
Not sure if it's already planned in the scope of this project, but maybe it's worth filing a documentation issue about updating the `Rebuild the indexes` section of the docs to include these new arguments.

Reply:
Yes, good point!

Reply:
I noticed you filed https://github.com/archivematica/Issues/issues/1254 Thanks!